tests/inc/health-check-curl-version-test.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace Yoast\WP\SEO\Tests\Inc;
+
+use Brain\Monkey;
+use Mockery;
+use Yoast\WP\SEO\Tests\TestCase;
+
+/**
+ * Unit Test Class.
+ *
+ * @group health-check
+ */
+class WPSEO_Health_Check_Curl_Version_Test extends TestCase {
+
+	/**
+	 * @var Mockery\Mock|\WPSEO_Health_Check_Curl_Version
```

Comment:
Missing description.

Reply:
Not sure what description to use. Is it really needed? There are a few more in this PR and hundreds similar cases in the codebase.

Reply:
There is a PHP CS check on the way that looks for a prop description. For now it's fine, but at some point we will need it. I would name it something like:
```suggestion
         * The instance to test.
         *
	 * @var Mockery\Mock|\WPSEO_Health_Check_Curl_Version
```

Reply:
There is a PHP CS check on the way that looks for a prop description. For now it's fine, but at some point we will need it.